dpdk.spec

```diff
@@ -36,9 +36,6 @@ Source502: set_config.sh
 #Source505: ppc_64-power8-linuxapp-gcc-config
 Source506: x86_64-native-linuxapp-gcc-config
 
-# Patches only in dpdk package
-Patch1: v19.02...67b915b09.patch
-
```

Comment:
You need to provide the patch (https://github.com/spdk/dpdk/compare/v19.11...3fcb1dd395a870c64450f151a53338034b832906 for spdk 20.01.1) that the spdk team adds to dpdk which is basically these commits:

```
* 3fcb1dd39 - (HEAD) contigmem: cleanup properly when load fails (2 weeks ago) <Jim Harris>
* fdb511332 - config: disable build of NXP PFE PMD Driver on ARM (2 months ago) <Alexey Marchuk>
* a0c8f2041 - config: allow the shared library to be built. (3 months ago) <Seth Howell>
* 33918d7f2 - config: comment out vhost (3 months ago) <Darek Stojaczyk>
* 92c570e36 - config: comment out options that we want to override in the makefile (3 months ago) <paul luse>
* 5d0be1f14 - config: don't compile libs we don't use (3 months ago) <Dariusz Stojaczyk>
```